addons/base_import/static/src/legacy/js/import_menu.js

```diff
@@ -1,18 +1,17 @@
 odoo.define('base_import.ImportMenu', function (require) {
     "use strict";
 
-    const DropdownMenuItem = require('web.DropdownMenuItem');
     const FavoriteMenu = require('web.FavoriteMenu');
     const { useModel } = require('web.Model');
 
+    const { Component } = owl;
```

Comment:
[IMP] web: use new dropdown in legacy too

This commit also needs a bit more explanations of what we're doing, and why